addons/api/fnc_getVehicleRacksPreset.sqf

```diff
@@ -0,0 +1,25 @@
+/*
+ * Author: ACRE2Team
+ * Gets the preset name used for initialising the vehicle racks.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT> (Default: objNull)
+ *
+ * Return Value:
+ * Preset name. Empty string if not defined <STRING> 
+ *
+ * Example:
+ * [cursorTarget] call acre_api_fnc_getVehicleRacksPreset
+ *
+ * Public: Yes
+ */
+#include "script_component.hpp"
+
+params [["_vehicle", objNull], ["_presetName", ""]];
+
+if (isNull _vehicle) exitWith {
+    ERROR("Vehicle is null");
```

Comment:
I forget, doesn't this throw an on-screen error as well? Or is that just `ERROR_WITH_MESSAGE_n`?